service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -1202,35 +1190,46 @@ private void checkAmountOfBagsIfCourierLimitByAmountOfBag(TariffsInfo courierLoc
         }
     }
 
-    private long formBagsToBeSavedAndCalculateOrderSumClient(
-        Map<Integer, Integer> getOrderBagsAndQuantity) {
-        long sumToPayInCoins = 0L;
-
-        for (Map.Entry<Integer, Integer> temp : getOrderBagsAndQuantity.entrySet()) {
-            Integer amount = getOrderBagsAndQuantity.get(temp.getKey());
-            Bag bag = bagRepository.findById(temp.getKey())
-                .orElseThrow(() -> new NotFoundException(BAG_NOT_FOUND + temp.getKey()));
-            sumToPayInCoins += bag.getFullPrice() * amount;
-        }
-        return sumToPayInCoins;
+    private long calculateOrderSumWithoutDiscounts(List<OrderBag> getOrderBagsAndQuantity) {
```

Comment:
```suggestion
    private BigDecimal calculateOrderSumWithoutDiscounts(List<OrderBag> getOrderBagsAndQuantity) {
```

Reply:
Throughout the program we use Long for price. Therefore, I am not sure how appropriate it is to change it